docs/pages/repo/docs/core-concepts/monorepos/configuring-workspaces.mdx

```diff
@@ -0,0 +1,181 @@
+}
+```
+
+<Callout>
+  For now, the only valid value for the `extends` key is `["//"]`. The
+`//` syntax, you may recall, signifies "root", similar to how it's used to
+[declare root tasks][1].
+</Callout>
+
+Extended configuration in a workspace can reset or override any of [the
+configuration for a pipeline task][2]. If you don't include a key, it will inherit
+the value from the extended `turbo.json`.
+
+## Examples
+
+To illustrate, let's look at a couple of use cases.
+
+### Different Frameworks
+
+Let's say your monorepo has a many Next.js apps, and one Sveltekit app. By
+default, both frameworks create their build with the `build` script in their
+respective `package.json`s. With a single `turbo.json` you can configure this
+with:
+
+```jsonc filename="turbo.json"
+{
+  "pipeline": {
+    "build": {
+      "outputs": [".next/**", ".svelte-kit/**"],
+    }
+  }
+}
+```
+
+Notice how both `.next/**` and `.svelte-kit/**` need to be specified as outputs,
+even though though the Next.js apps do not generate this output. With Workspace
+overrides, you can instead add a new configuration at
+`apps/my-svelte-kit-app/turbo.json`, and remove the config from the root
+configuration:
+
+```jsonc filename="apps/my-svelte-kit-app/turbo.json"
+{
+  "extends": ["//"],
+  "pipeline": {
+    "build": {
+      "outputs": [".svelte-kit/**"]
+    }
+  }
+}
+```
+
+```diff filename="turbo.json"
+{
+  "pipeline": {
+    "build": {
+-      "outputs": [".next/**", ".svelte-kit/**"]
++      "outputs": [".next/**"]
+    }
+  }
+}
+```
+
+### Specialized Tasks
+
+In another example, say that the `build` task in one workspace `dependsOn` a
+`compile` task. You could universally declare it as `dependsOn: ["compile"]`.
+This means that your root `turbo.json` has to have a empty `compile` task entry:
+
+```json filename="turbo.json"
+{
+  "extends": ["//"],
+  "pipeline": {
+    "build": {
+      "dependsOn": ["compile"]
+    },
+    "compile": {}
+  }
+}
+```
+
+With a Workspace override, you can move that `compile` task into the
+`apps/my-custom-app/turbo.json` and remove it from the root.
+
+## Relationship to `Workspace#Task` syntax
+
+You may be wondering: how does this relate to [`workspace#task` entries][3] in the root turbo.json?
+These features are very similar, with one major difference. When you declare a specific workspace task
+in the root turbo.json, it is _completely_ preferred over the baseline task configuration. With a
+Workspace Override, the task configuration is merged. Consider the example of the monorepo with
+four Next.js apps and a Sveltekit app again. Without a workspace override, you may configure your
+root `turbo.json` like this:
+
+```json filename="turbo.json"
+{
+  "pipeline": {
+    "build": {
+      "outputMode": "hash-only",
+      "outputs": [".next/**"],
+    },
+    "my-sveltekit-app#build": {
+      "outputs": [".svelte-kit/**"]
+    }
+  }
+}
+```
+
+This works, but it's important to understand that the `my-sveltekit-app#build`
```

Comment:
Picking up from comment above, seeing how I can do more with less...

"It might have snuck by us that our `my-sveltekit-app#build` task lost the `outputMode` when we really were only meaning to change the cache outputs. We would need to duplicate the `outputMode` key get that configuration back.

If we have many configurations, this ends up being a lot of duplicated code to maintain. With Workspace Configs, you don't need to duplicate any configuration. Instead, you only need to override what needs to be changed."

Reply:
reworded to talk more about the duplication